exercises/handling-errors/practice/src/worker.ts

```diff
@@ -0,0 +1,21 @@
+import * as activities from './activities';
+import { TASK_QUEUE_NAME } from './shared';
+import { NativeConnection, Worker } from '@temporalio/worker';
+
+async function run() {
+  const connection = await NativeConnection.connect({ address: 'localhost:7233' });
```

Comment:
nit: Part of me, always wants to mention that 7233 is the default gRPC address for the Temporal CLI server.